docs/versioned-plugins/codecs/cef-v4.1.3.asciidoc

```diff
@@ -12,7 +12,7 @@ START - GENERATED VARIABLES, DO NOT EDIT!
 END - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
 
-[id="plugins-{type}-{plugin}"]
+[id="{version}-plugins-{type}s-{plugin}"]
```

Comment:
This problem is caused by bad asciidoc in older versions of the plugin. We can fix it with a simple search/replace in the old files.